Makefile

```diff
@@ -13,7 +13,7 @@ install:
 
 check:
 	find . -name "*.pyc" -exec rm {} \;
-	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=yes --verbose --showlocals --cov=packit --cov-report=term-missing $(TEST_TARGET)
```

Comment:
what went wrong with the colors?

Reply:
see eg log: https://softwarefactory-project.io/logs/35/435/8f5c42034e285864cc1d4c7d0c3eabbb80974a96/check/tests-rpm/8c090a8/job-output.txt.gz
there are these strange escape sequences:
```
2019-07-29 08:05:59.588161 | test-node | [1m>       assert result.exit_code == 0[0m
2019-07-29 08:05:59.588173 | test-node | [1m[31mE       assert 2 == 0[0m
2019-07-29 08:05:59.588236 | test-node | [1m[31mE        +  where 2 = <Result SystemExit(2)>.exit_code[0m
```

so I decided to remove it to not confuse logs with this.